modules/databend/examples_test.go

```diff
@@ -0,0 +1,89 @@
+package databend_test
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/databend"
+)
+
+func ExampleRun() {
+	ctx := context.Background()
+
+	databendContainer, err := databend.Run(ctx,
+		"datafuselabs/databend:v1.2.615",
+		databend.WithUsername("databend"),
+		databend.WithPassword("databend"),
+	)
+	defer func() {
+		if err := testcontainers.TerminateContainer(databendContainer); err != nil {
+			log.Printf("failed to terminate container: %s", err)
+		}
+	}()
+	if err != nil {
+		log.Printf("failed to start container: %s", err)
+		return
+	}
+	// }
+
+	state, err := databendContainer.State(ctx)
+	if err != nil {
+		log.Printf("failed to get container state: %s", err)
+		return
+	}
+
+	fmt.Println(state.Running)
```

Comment:
bug: missing output definition to ensure the example validates

Reply:
Fixed.

Reply:
We need here something like this for the testable example to be runnable by the test runner:
```suggestion
	fmt.Println(state.Running)

	// Output:
	// true
```

Reply:
Got it and fixed.